src/main/java/net/atomique/ksar/UI/GraphView.form

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.1" encoding="UTF-8" ?>
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
This change was suggested by NetBeans, and Internet says that 1.1 is almost never used.